metricflow/model/transformations/convert_median.py

```diff
@@ -28,4 +28,5 @@ def transform_model(model: UserConfiguredModel) -> UserConfiguredModel:  # noqa:
                             )
                     measure.agg_params.percentile = MEDIAN_PERCENTILE
                     measure.agg_params.use_discrete_percentile = False
+                    # let's not set use_approximate_percentile to be false due to valid performance reasons
```

Comment:
Do we need to assert that approx is set to something?

Also, we missed the validation on discrete - people shouldn't be asking for discrete medians, right?